modules/secured-cluster-configuration-options-operator.adoc

```diff
@@ -79,8 +79,7 @@ a| Use one of the following values to specify if the admission controller must c
 The default value is `DoNotScanInline`.
 
 | `admissionControl.timeoutSeconds`
-| Use this parameter to specify the maximum number of seconds {product-title} must wait for an admission review before marking it as fail open.
-
+| Use this parameter to specify the maximum number of seconds {product-title-short} must wait for an admission review before marking it as fail open. Beginning in release 4.5, Red{nbsp}Hat reduced the default timeout setting for the {product-title-short} admission controller webhooks from 20 seconds to 10 seconds, resulting in an effective timeout of 12 seconds within the `ValidatingWebhookConfiguration`. This change does not affect {ocp} users because {ocp} caps the timeout at 13 seconds. However, for Kubernetes clusters, image scanning and pulling that are done as part of the webhook execution might require you to configure a longer timeout value.
```

Comment:
I am not sure if the word `must` here (as in "{product-title-short} must wait") is a good fit.
I mean, this this a timeout setting. So, it is a guarantee for the waiting time to not exceed the configured timeout. But it is not the case that ACS "must" wait for such a duration(?). (I know that this is also in the old version)